src/commcare_cloud/terraform/README.md

```diff
@@ -64,163 +64,3 @@ You can run terraform with
 ```
 cchq <env> terraform apply
 ```
```

Comment:
@dannyroberts Should we add details about setting up SSM here or is it automatically configured when a new account is created?

Reply:
Great question. That is indeed set up manually-ish in the console, but may also be partially confiured by aws control tower. I don't think we need to proactively document, but may be worth it next time we happen to run through it. Feel free to offline if you want to explore more